src/base/bittorrent/session.cpp

```diff
@@ -63,7 +63,9 @@
 #include <QFile>
 #include <QHostAddress>
 #include <QNetworkAddressEntry>
+#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
 #include <QNetworkConfigurationManager>
```

Comment:
I purposely don't want to move all these conditional inclusions out of their original places to keep the changes extremely self-explanatory.